cypress/e2e/api/api.cy.js

```diff
@@ -0,0 +1,109 @@
+/// <reference types="Cypress"/>
+
+const API_URL = Cypress.env('API_URL')
+
+describe('Verificar API', () => {
```

Comment:
Que tal algo mais descritivo?

```suggestion
describe('EngageSphere Customers API', () => {
```